### ISSUE TITLE
[TODO] Refactor the login page

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/RadioLogger#79]
Refactor the login, register and reset password functionality and its UI.

I've already added the new login page.

**Comment by jzunigarce** [comment]
Where is the register page?

**Comment by gbowne1** [comment]
I took a break. I will push that up here in a bit. @jzunigarce

**Comment by gbowne1** [comment]
Pushed up the new profile html.

**Comment by anubhav1206** [comment]
Can I work on this?

**Comment by jzunigarce** [comment]
I current work with login and register @gbowne1

**Comment by jzunigarce** [comment]
I'm going to fix the cors errors

**Comment by jzunigarce** [comment]
I upload and accept (By mistake) the PR. Please check the changes. See PR #80  for changes files

**Comment by jzunigarce** [comment]
@gbowne1 i auto-accept(by mistake) a PR #80. can you please review it?

**Comment by gbowne1** [comment]
@jzunigarce I am going to remove the excess scripts from login.html and from register.html  and push up that change.  The

**Comment by jzunigarce** [comment]
<img width="948" alt="image" src="https://github.com/gbowne1/RadioLogger/assets/722001/80ece46a-1daf-4aa9-9fdd-2d6037515f3e">
<img width="1037" alt="image" src="https://github.com/gbowne1/RadioLogger/assets/722001/54d0296e-38e2-4871-a8d9-14b370a22cad">
This are the last changes in the repo

**Comment by jzunigarce** [comment]
@gbowne1 Download the changes to your repo

**Comment by gbowne1** [comment]
yes @jzunigarce I have synced the changes. It now exists as above. I just pushed up a change to remove all the unnecessary scripts in the login.html and the register.html (the ones in the commented out section mostly)

**Comment by gbowne1** [comment]
this comment is mostly for me to remember what we need for login form, but this is the refactor we need.

- Username: This field allows the user to enter their username associated with their account.
 
- Password: This field is used for entering the password associated with the username or email. This will have a show password eye icon to toggle the password visibility.

- Remember Me: This checkbox allows the user to choose whether they want the system to remember their login credentials for future visits. 

- Forgot or Reset Password: This link provides a way for users to reset their password if they have forgotten it.  we already    have the script to do this.

- Sign In/Log In Button: This button triggers the authentication process and logs the user into the system.  Just needs renaming the button.
